src/quo/components/tags/collectible_tag/view.cljs

```diff
@@ -1,69 +1,71 @@
 (ns quo.components.tags.collectible-tag.view
   (:require
+    [oops.core :as oops]
     [quo.components.icon :as icons]
     [quo.components.markdown.text :as text]
     [quo.components.tags.collectible-tag.style :as style]
     [quo.theme]
     [react-native.core :as rn]
     [react-native.hole-view :as hole-view]
+    [reagent.core :as reagent]
     [schema.core :as schema]))
 
 (def ?schema
   [:=>
    [:catn
     [:props
      [:map {:closed true}
-      [:options {:optional true} [:maybe [:enum false :add :hold]]]
-      [:size {:optional true} [:enum :size-24 :size-32]]
-      [:blur? {:optional true} :boolean]
+      [:options {:optional true} [:maybe [:enum :add :hold]]]
+      [:size {:optional true} [:maybe [:enum :size-24 :size-32]]]
+      [:blur? {:optional true} [:maybe boolean?]]
       [:theme :schema.common/theme]
-      [:collectible-img-src [:or :int :string]]
-      [:collectible-name :string]
-      [:collectible-id :string]
-      [:container-width :number]
-      [:on-layout {:optional true} [:maybe fn?]]]]]
+      [:collectible-img-src :schema.common/image-source]
+      [:collectible-name string?]
+      [:collectible-id string?]]]]
```

Comment:
Same here, why `string?` over `:string` ? Most schemas that are already merged prefer the `:keyword` variant.